script/foreman-debug

```diff
@@ -125,6 +125,9 @@ add_files() {
       add_file $FILE
       SUMSIZE=$(($((SUMSIZE))+$((SIZE))))
     fi
+    touch -c -r "$FILE"   "$DIR$FILE"
+    touch -c -r "$FILE"   "$DIR$FILE.txt"
+    touch -c -r "$SUBDIR" "$DIR$SUBDIR"
```

Comment:
Hello, thanks. Please move the touch commands where the files gets created. These lines are with `mkdir` and `cat` in `add_file` because this function is often used as well.

The third touch is not necessary, the `cp` can be given argument to preserve timestamp.